webperf-getting-started/Getting started with web performance.md

```diff
@@ -0,0 +1,184 @@
+
+Your landing page is a first natural candidate. If you're a newspaper, a selected popular  past article will be another one.
+
+For example, Wikimedia's performance team uses "Barack Obama" and "Sweden" articles from English Wikipedia in their benchmarks and performance monitoring.
+
+
+# Get to know the tools
+
+Run your page in several performance tools, to get familiar with them, and identify some "quick wins". Each of those tools can provide some insights that others may be missing, or can explain it better, so it's worth trying them all to see which ones are most useful for your particular use cases.
+
+- **Lighthouse** is probably the easiest to get started, provides lots of details relevant for modern JS-heavy web applications: run the audit locally via "Audits" tab in Chrome Dev Tools, or from any browser [via a webapp](https://developers.google.com/web/tools/lighthouse/run) or [via WPT](https://www.webpagetest.org/lighthouse).
+- **[WebPageTest](https://www.webpagetest.org/)** is the "Swiss Army knife" of web performance, very powerful and with many "hidden features" for advanced users. The more you dive into webperf, the more you will be using it. [Using WebPageTest](http://shop.oreilly.com/product/0636920033592.do) book is a great reference.
+- **[Yellow Lab Tools](https://yellowlab.tools/)** can also provide a useful breakdown of various performance metrics.
+- While not strictly performance-focused, check also **[WebHint](https://webhint.io/scanner/)** and **[RedBot](https://redbot.org/)** which can uncover interoperability and correctness issues of your website, suggest improvements, and help you get rid of useless HTTP headers. 
+- Check also **[SiteSpeed.io](https://www.sitespeed.io/)** for a vast collection of open source webperf tools.
+
+At this point, you will already know some things that are relatively easy to get fixed (at least in theory), but have high impact: uncompressed assets, unoptimized images etc. 
+
+
+# Know your HTML
+
+Get your website's HTML output (`Ctrl-U` in Firefox and Chrome, for example), unminify, and analyze it. **Create a high level documentation of what makes it into your HTML, why it's there** (ask your teammates if you don't know), and how many bytes it takes.
+
+On long-living projects, it might be the case that some cruft has accumulated over the years that is not really needed anymore, and can be safely removed. (bonus points if it's a third-party JavaScript!)
+
+
+# Know your subresources
+
+The next thing to understand your website is to know what HTTP requests a page load triggers, when, and why. It might be helpful to run a WebPageTest test, copy [the list of all the requests (at the bottom)](https://www.webpagetest.org/result/181216_GB_9b0d2e1a261ee5401dfd19842cb74570/1/details/#waterfall_view_step1) to a spreadsheet, and analyze the role of each them (particularly the first 10-20 on the list), and how critical are they.
```

Comment:
```suggestion
 The next thing to understand your website is to know what HTTP requests a page load triggers, when, and why. It might be helpful to run a WebPageTest test, copy [the list of all the requests (at the bottom)](https://www.webpagetest.org/result/181216_GB_9b0d2e1a261ee5401dfd19842cb74570/1/details/#waterfall_view_step1) to a spreadsheet, and analyze the role of each them (particularly the first 10-20 on the list), and how critical they are.
```